### ISSUE TITLE
Problems with newlines/lists

### ISSUE THREAD

**Issue by Nebucatnetzer** [deathau/markdownload#31]
I'm experiencing some problems with new lines in lists.
For some reason obsidian doesn't recognise the new lines made by Markdownload.
Example:
```
- meta:
    - topics: [[to_categorise]]
    - url: https://www.derbund.ch/kanton-bern-hat-sich-bei-neuem-polizeizentrum-verrechnet-453109101204
    - date: [[2020-11-14]]
    
- meta:
    - topics: [[to_categorise]]
    - url: 
    - date: [[2020-11-14]]
------
```

![Screenshot from 2020-11-14 13-57-54](https://user-images.githubusercontent.com/2287221/99147489-a40ec480-2681-11eb-82f4-985a362b1a5b.png)

As you can see the raw text looks identical but the one generated by Markdownload doesn't get processed correctly.
I tried to copy the working code into Markdownload but the output still doesn't work.
Interestingly the line endings seem to be the same but the tabs are not, in VSCode they look different even though they feel both like they are 4 spaces:

![Screenshot from 2020-11-14 14-04-15](https://user-images.githubusercontent.com/2287221/99147576-6d857980-2682-11eb-9c40-16e476eba08d.png)

Maybe it's a special type of space?
https://www.fileformat.info/info/unicode/char/search.htm?q=%C2%A0&han=Y&preview=entity
![Screenshot from 2020-11-14 14-08-40](https://user-images.githubusercontent.com/2287221/99147645-f2709300-2682-11eb-9281-2486f731959f.png)

**Comment by deathau** [comment]
In Obsidian there's a setting for "strict line breaks" you might need to toggle

A quirk of pure markdown is that a line break has to be two spaces followed by a new line, otherwise it counts as being on the same line. If you do two new lines it counts as a new paragraph.

**Comment by Nebucatnetzer** [comment]
No that is not the problem (and it is actually a feature of Markdown I quite like :) ).
When I enable the strict line breaks it looks better but Obsidian still doesn't recognise it as a proper list.

![Screenshot from 2020-11-16 11-48-34](https://user-images.githubusercontent.com/2287221/99244289-2b834180-2802-11eb-9308-f8e1ae20e431.png)

As you can see here, even Github has problems with the output from Markdownload.
The interesting things is, that I only see this problem with the downloaded Markdown file, the output in the popup seems to be fine.

**Example**

- meta:
    - topics: [[to_categorise]]
    - url: https://docs.scrapy.org/en/latest/topics/broad-crawls.html#enable-crawling-of-ajax-crawlable-pages
    - date: [[2020-11-16]]
    
- meta:
    - topics: [[to_categorise]]
    - url: https://docs.scrapy.org/en/latest/topics/broad-crawls.html#enable-crawling-of-ajax-crawlable-pages
    - date: [[2020-11-16]]

Code of the example

```
- meta:
    - topics: [[to_categorise]]
    - url: https://docs.scrapy.org/en/latest/topics/broad-crawls.html#enable-crawling-of-ajax-crawlable-pages
    - date: [[2020-11-16]]
    
- meta:
    - topics: [[to_categorise]]
    - url: https://docs.scrapy.org/en/latest/topics/broad-crawls.html#enable-crawling-of-ajax-crawlable-pages
    - date: [[2020-11-16]]
```

**Comment by deathau** [comment]
I can't seem to replicate this, but the next version will be using a small CodeMirror instance instead of a regular text box, and I'd love to hear if it resolves this issue.

**Comment by Nebucatnetzer** [comment]
Sure I'll give it a try. I'm anyway already looking forward to the new release because it fixes #40 
Thank you for continue working on this, it's currently my most used extension besides the privacy/adblocker ones.

**Comment by Nebucatnetzer** [comment]
It's working now but I had to reenter the spaces in order to have it work correctly.
